sycl/doc/extensions/experimental/sycl_ext_oneapi_complex.asciidoc

```diff
@@ -248,74 +249,155 @@ public:
 
   /* ... */
 
-  friend marray operator %(const marray &lhs, const marray &rhs) = delete;
-  friend marray operator %(const marray &lhs, const value_type &rhs) = delete;
-  friend marray operator %(const value_type &lhs, const marray &rhs) = delete;
+  /// Adds and assigns marray rhs to marray lhs.
```

Comment:
I think the following phrasing is a little clearer:
```suggestion
  /// Adds rhs to lhs and assigns the result to lhs.
```

Reply:
I understand that this should be propagated to the rest of the operator descriptions.

I also understand that you removed the  `marray` part because it is enough explicit to understand the `lhs` and `rhs` types from the prototype.

What about the operator overloads that takes a marray on the `lhs` and a complex on the `rhs`.
Should we still use the `rhs to lhs` phrasing? Not `complex number rhs to lhs` ?

Reply:
Gentle ping in case you missed this comment @steffenlarsen

Reply:
Does the clarification commit resolves your comment @steffenlarsen

Reply:
I'd say so, thanks!